lib/config/router.go

```diff
@@ -0,0 +1,13 @@
+package config
+
+// router.config options
+type RouterConfig struct {
+  NetDbDir string
```

Comment:
I would ideally like to use java i2p's network database filesystem skiplist, i2pcpp tried sqlite and it turned out it wasn't efficient at all.